README.md

```diff
@@ -1,2 +1,233 @@
+
+```python
+>>> pyminio_client.mkdirs('/foo/bar/baz/')
+```
+
+### listdir
+Pyminio.listdir will return the directory's content as a list of directorys and files name. Works like os's 'listdir'.
+
+This function must get a directory path or it will raise a ValueError.
+
+```python
+>>> pyminio_client.listdir('/foo/bar/baz/')
+['file_name_1', 'file_name_2', 'directory_name/']
+```
+
+There is an option to use the only_files flag to get only files from listdir.
+
+```python
+>>> pyminio_client.listdir('/foo/bar/baz/', only_files=True)
+['file_name_1', 'file_name_2']
+```
+
+### exists
+Pyminio.exists will return a boolean that confirm reather this path exists or not in the server. Works like os.path.exists.
+
+```python
+>>> pyminio_client.exists('/foo/bar/baz/file_name_1')  # existed file
+True
+>>> pyminio_client.exists('/foo/bar/baz/file_name_3')  # not existed file
+False
+>>> pyminio_client.exists('/all/path/wrong/')  # not existed path
+False
+```
+
+### isdir
+Pyminio.isdir will return True only if the given path exists and is a directory. Works like os.path.isdir
+
+```python
+>>> pyminio_client.isdir('/foo/bar/baz/file_name_1')  # existed file
+False
+>>> pyminio_client.isdir('/foo/bar/baz/directory_name/')  # existed directory
+True
+>>> pyminio_client.isdir('/all/path/wrong/but/directory/')  # not existed directory
+False
+```
+
+### truncate
+Pyminio.truncate will delete all minio's content.
+
+```python
+>>> pyminio_client.truncate()
+```
+
+### rmdir
+Pyminio.rmdir will delete the specified directory. Works like linux's 'rmdir (-r)'.
+
+It will raise a DirectoryNotEmptyError if given directory is not empty, except if the recursive flag is on and then it will delete given directory's path recursively.
+
+This function must get a directory path or it will raise a ValueError.
+
+```python
+>>> pyminio_client.rmdir('/foo/bar/baz/directory_name/')  # empty directory
+>>> pyminio_client.rmdir('/foo/bar/')  # unempty directory
+DirectoryNotEmptyError: can not recursively delete unempty directory
+>>> pyminio_client.rmdir('/foo/bar/', recursive=True)
+```
+
+### rm
+Pyminio.rm works like [rmdir](#rmdir) only that it can delete files too. Works like linux's rm (-r).
+
+```python
+>>> pyminio_client.rm('/foo/bar/baz/file_name')
+```
+
+### cp
+Pyminio.cp will copy one file or directory to given destination. Works like linux's 'cp (-r)'.
+
+This func can only copy recursively when the recursive flag is True. If not, it will raise a ValueError.
+
+#### How will the copy accure? (all directories are copied recursively in this examples)
+| src path   | dst path  | dst exists | new dst      | Explain                                                                |
+| ---------- | --------- | ---------- | ------------ | ---------------------------------------------------------------------- |
+| /foo/bar   | /foo/baz  |    ---     | /foo/baz     | The file's name will be changed from bar to baz.                       |
```

Comment:
What? Then it is move and not copy
You mean that the file's content will be moved to `baz` as well?